addons/project/models/project.py

```diff
@@ -2504,6 +2504,7 @@ def action_project_sharing_view_parent_task(self):
             return {}
         action = self.action_open_parent_task()
         action['views'] = [(self.env.ref('project.project_sharing_project_task_view_form').id, 'form')]
+        action['search_view_id'] = [self.env.ref("project.project_sharing_project_task_view_search").id]
```

Comment:
Cannot we just pass the id instead of a list? 🧐 
```suggestion
        action['search_view_id'] = self.env.ref("project.project_sharing_project_task_view_search").id
```

Reply:
done :)